### ISSUE TITLE
static file for lower zoom and serve dynamically for higher zoom

### ISSUE THREAD

**Issue by lhapaipai** [t-rex-tileserver/t-rex#256]
I wonder if if possible to pre-regenerate the tiles with a low zoom, those which are generally the most supplied and therefore take a long time to load and on the other hand to leave to the server the care of dynamically serving the tiles with high zooms ?

I am thinking of this option with the cache.file
we will specify in the cache configuration a max zoom, we are launching a generation of tiles , and after starting the server.
for each request, it checks if it has the cached file otherwise it generates it.

Thanks.

**Comment by pka** [comment]
This makes absolutely sense and looks like a simple but useful enhancement.

**Comment by tobwen** [comment]
Yeah, that would be nice. Right now, I'm doing this in the reverse proxy. Small-scale tiles are pre-generated and go to my "static" backend, while large-scale tiles get rendered dynamically.

Would be nice to wrap this up in the backend.

**Comment by pka** [comment]
The `cache_limits` settings in https://t-rex.tileserver.ch/doc/reference/#tileset should do this!
